go.mod

```diff
@@ -21,103 +21,95 @@ require (
 	github.com/hashicorp/go-version v1.6.0
 	github.com/matryer/is v1.4.0
 	github.com/vishvananda/netlink v1.1.0
-	github.com/wailsapp/wails/v2 v2.3.1
+	github.com/wailsapp/wails/v2 v2.2.0
```

Comment:
any reason for a downgrade here?

Reply:
rectified. had to update the wails cli so it doesnt auto downgrade